hedyorg/hedy
# Fix keypress modal not showing

Fixes #5745

**How to test**
Open level 5 and execute the following code. Check that in the output window there is a modal that indicates when the program is waiting for input. Once you press a button, the modal should disappear.
```
if y is pressed forward 15
else turn 90
```
